### ISSUE TITLE
lab mr create throws  "mrCreate.go:55: gitlab project not found"

### ISSUE THREAD

**Issue by jonathancolby-olx** [zaquestion/lab#160]
#### command and error:

```
$lab mr create
2018/06/07 23:32:03 mrCreate.go:55: gitlab project not found
```

#### info 
OSX High Sierra 10.13.4 (17E202)

install method:  `brew install zaquestion/tap/lab`

version:
```
lab version
git version 2.15.0
hub version 2.2.9
lab version 0.11.1
```

gitlab enterprise, self-hosted.

 origins:

```
git remote -v
origin  git@git.naspersclassifieds.com:infrastructure/europe/terraform-olxeu-infrastructure.git (fetch)
origin  git@git.naspersclassifieds.com:infrastructure/europe/terraform-olxeu-infrastructure.git (push)
```

**Comment by zaquestion** [comment]
@jonathancolby-olx This look valid to me -- If you try a `lab mr list` do you also get no results?

**Comment by zaquestion** [comment]
@jonathancolby-olx We've also seen this error when the branch contains a character that would be url encoded and the user is running gitlab behind apache without proper configuration. 
https://github.com/zaquestion/lab/issues?q=is%3Aissue+label%3Aapache+is%3Aclosed

**Comment by jonathancolby-olx** [comment]
@zaquestion - hi. thanks for looking at my issue.  it seems even list is failing.  

```
lab mr list
2018/06/08 10:46:02 mrList.go:38: gitlab project not found
```

have people used the tool on self-hosted gitlab installations?  our project doesnt have any funny characters `https://git.naspersclassifieds.com/infrastructure/europe/terraform-olxeu-infrastructure`

**Comment by zaquestion** [comment]
@jonathancolby-olx Are you still having issues?

**Comment by zaquestion** [comment]
@jonathancolby-olx Sorry for the skepticism, I've seen this issue reported a few times over the lab's lifespan and to date it has exclusively been user error and they've figured it out pretty quick.

One of my coworkers actually ran into the issue and I was able to debug it with them. The issue occurs when the gitlab host is configured with a trailing slash.
Like so.
```
"core" = {
  "host" = "https://gitlab.com/"
  "token" = "<redacted>"
  "user" = "zaq"
}
```
It can be worked around by removing the trailing slash `"host" = "https://gitlab.com"` -- I'll get a fix in the next release, this used to work fine, but something changed.

**Comment by jonathancolby-olx** [comment]
Hello @zaquestion -

Sorry for the delayed response to your question.  I appreciate you looking into this.  I will need to test again within the next day or so and I will post whether the recommended workaround fixed the problem.

**Comment by zaquestion** [comment]
@jonathancolby-olx A fix has been released in `0.12.0` as well.
